docs/guides/schema.md

```diff
@@ -4,54 +4,61 @@ import {InlinePlayground} from '../../src/components/InlinePlayground';
 
 ## What is a schema?
 
-The **Schema** of a Permissions System defines the types of objects found in the permission, how those objects relate to one another, and the permissions that can be computed off of those relations.
+The **Schema** of [SpiceDB] or a Permissions System on [Authzed] defines the types of objects found, how those objects relate to one another, and the permissions that can be computed off of those relations.
 
 Examples of object types include resources, users, groups, documents, or any kind of object either being protected or for whom something is being protected.
 
 For the exhaustive description of the language used to write schemas, check out the [schema language reference].
 
 [schema language reference]: /reference/schema-lang
+[SpiceDB]: https://github.com/authzed/spicedb
+[Authzed]: https://app.authzed.com
 
-:::info
-This guide uses a Permissions System named example.
+:::warning
+This guide does not specify prefixes on its definitions. SpiceDB and the playground tool accept schema without prefixes, but they are required on Authzed.
```

Comment:
Do we want to refer to it as Authzed.com everywhere? I suspect it might be easier to call everything SpiceDB and just call Authzed.com a managed SpiceDB rather than treat it like it's some how fundamentally different.

Reply:
We should at least mention it at the beginning and in this initial warning. I went with `Authzed` to allow it to cover on-prem as well